Description: disable deprecated protocols
Author: Yadd <yadd@debian.org>
Bug-Debian: https://bugs.debian.org/1011852
Forwarded: no
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2022-06-01

--- a/index.js
+++ b/index.js
@@ -9,7 +9,6 @@
   'des-ede-cbc': des.CBC.instantiate(des.EDE),
   'des-ede': des.EDE,
   'des-cbc': des.CBC.instantiate(des.DES),
-  'des-ecb': des.DES
 }
 modes.des = modes['des-cbc']
 modes.des3 = modes['des-ede3-cbc']
--- a/modes.js
+++ b/modes.js
@@ -1,11 +1,3 @@
-exports['des-ecb'] = {
-  key: 8,
-  iv: 0
-}
-exports['des-cbc'] = exports.des = {
-  key: 8,
-  iv: 8
-}
 exports['des-ede3-cbc'] = exports.des3 = {
   key: 24,
   iv: 8
